brands/amenity/cinema.json

```diff
@@ -80,6 +80,17 @@
       "short_name": "Century"
     }
   },
+  "amenity/cinema|CGR": {
+    "countryCodes": ["fr"],
+    "matchNames": ["cgr"],
```

Comment:
This line can go away, matching is case-insensitive.
```suggestion
```

Reply:
Ok, I didn't know the case-insensitive for matching brands. I will correct this.